source/compatibility.tex

```diff
@@ -159,24 +158,24 @@
 \effect
 Change to semantics of well-defined feature.
 \difficulty
-Semantic transformation
+Semantic transformation.
 \howwide
-Seldom
+Seldom.
 
 \ref{basic.start}
-\change Main cannot be called recursively and cannot have its address taken
+\change The \tcode{main} function cannot be called recursively and cannot have its address taken.
```

Comment:
I would even delete the word "recursively". But I suppose this is non-normative, so no big deal.